include/seqan3/alignment/pairwise/align_pairwise.hpp

```diff
@@ -38,10 +38,92 @@
 namespace seqan3
 {
 
+/*!\brief Computes a pairwise alignment over a sequence pair or a range of sequence pairs.
+ * \ingroup pairwise_alignment
+ *
+ * \tparam sequence_t         The type of sequence pairs. Either a type modelling seqan3::tuple_like_concept
+ *                            with a tuple size of exactly two or std::ranges::InputRange whose value type models the
+ *                            before mentioned tuple concept.
+ * \tparam alignment_config_t The type of the alignment configuration; must be a seqan3::configuration.
+ * \param[in] seq             A tuple with two sequences or an input range over such tuples.
```

Comment:
Is it really just InputRange?

Reply:
It really is. The range that provides the sequence only needs to be iterated once.